examples/kustomization/operator-external-idp-oid/README.md

```diff
@@ -0,0 +1,95 @@
+# Operator Console SSO with OpenID
+
+Operator Console supports authentication with a Kubernetes Service Account Json Web Token (JWT) or OpenID. This guide explains how to configure OpenID authentication for Operator Console using the [OpenID Authorization Code Flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth).
+
+Note: only one authentication method can be enabled at the same time, either JWT or OpenID.
```

Comment:
```suggestion
Note: only one authentication method can be enabled at a time, either JWT or OpenID.
```
@ravindk89 ^^^ ?

Reply:
we can skip this for now to unlock release to users, @feorlen is going to work on better wording in documentation